LeadAlerts.Web/Domain/Configuration.cs

```diff
@@ -6,12 +6,14 @@ public class Credentials
     {
         public static string AccountSID
         {
-            get { return WebConfigurationManager.AppSettings["TwilioAccountSid"]; }
+            get { return WebConfigurationManager.AppSettings["TwilioAccountSid"] ??
+                    "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"; }
```

Comment:
Ah yes, this is a vestige from when we I had attempted to test the twilio api calls directly instead of mocking the client wrapper. I should remove this...